module/importer/migrations/Version20180618134343.php

```diff
@@ -52,7 +52,7 @@ class VARCHAR(255) NOT NULL,
         $this->addSql(
             'ALTER TABLE importer.import
              ADD CONSTRAINT import_source_fk FOREIGN KEY (source_id) 
-             REFERENCES importer.source(id)  ON UPDATE CASCADE ON DELETE RESTRICT'
+             REFERENCES importer.source(id)  ON UPDATE CASCADE ON DELETE CASCADE'
```

Comment:
You know that it will not work on current implementations ?

Reply:
add to new migration